src/header-navigation/index.js

```diff
@@ -0,0 +1,16 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+export {default as HeaderNavigation} from './header-navigation';
+// Styled elements
+export {
+  Root as StyledRoot,
+  NavigationItem,
+  NavigationList,
```

Comment:
I wonder if we should create container components for both `NavigationItem` and `NavigationList` that will take overrides and render `StyledNavigationItem` and `StyledNavigationList` accordingly. Since this falls outside the pattern we used in other components so far. Same as we do for the `HeaderNavigation`. 
@schnerd what do you think about it?

Reply:
If we want to export it as a container we should add a container component to it

Reply:
fixed